### DISCUSSION TITLE
Internal kiosk dashboards

### DISCUSSION THREAD

**Discussion by kevinsimper** [grafana/grafana#61668] [Dashboards, Panels & Visualizations]
It would be cool if we can show dashboards on internally screens without making them public.

The displays are using a device that does not allow user input, so the kiosk mode should be non-interactive.

It could either be a personal key per display or IP limited.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I feel like you could configure grafana to work within an intranet to have this be possible. Feel free to reach out to our[ community forum](https://community.grafana.com/) to see if others have configured Grafana in this way and  please make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
